webpack/helpers.js

```diff
@@ -58,24 +59,73 @@ function getConfig(projectDir, proxyUrl, projectPathConfig, assetsPathConfig = '
     applicationAdminEntry: path.resolve(absolutePath, assetsPathConfigClean, 'application-admin.js'),
     applicationBlocksEntry: path.resolve(absolutePath, blocksAssetsPathConfigClean, 'application-blocks.js'),
     applicationBlocksEditorEntry: path.resolve(absolutePath, blocksAssetsPathConfigClean, 'application-blocks-editor.js'),
+
+    blocksManifestSettingsPath: path.resolve(absolutePath, blocksManifestSettingsPathClean),
   };
 }
 
 /**
- * Generate project paths for node_modules and libs root path.
+ * Convert Recursive Json data only for colors to SASS valid output.
  *
- * @param string nodeModules Path to node modules in general it is __dirname.
+ * @param object Json data only for colors.
+ */
+function convertJsonColorsToSass(data) {
+  let output = '';
+
+  for (const property in data) {
+    if (data.hasOwnProperty(property)) {
+
+      if (typeof data[property] === 'string' && property === 'color') {
+        output += data[property];
+      }
+
+      if (typeof data[property] === 'object') {
+        output += `${data[property].slug}: ${convertJsonColorsToSass(data[property])},`;
+      }
+    }
+  }
+
+  return output;
+}
+
+/**
+ * Convert Recursive Json data to SASS valid output.
  *
- * @since 2.0.0
+ * @param object Json data to convert.
```

Comment:
```suggestion
 * @param object Json data to convert.
 * 
 * @since 3.1.2
```
Ne pise od kojeg releasea.

Reply:
we will be removing all this since tags.